sound/soc/sof/Makefile

```diff
@@ -3,7 +3,7 @@
 ccflags-y += -DDEBUG
 
```

Comment:
@ranj063 commit message says that MFD client is registered under the SOF PCI device. Shouldn't this be generic? For example on IMX DSP doesn't sit on a PCI bus.

Reply:
@dbaluta ack. will make the commit message generic.